group/edwards25519/scalar.go

```diff
@@ -4,11 +4,196 @@
 
 package edwards25519
 
+import (
+	"crypto/cipher"
+	"errors"
+	"io"
+	"math/big"
+
+	"gopkg.in/dedis/kyber.v1"
+	"gopkg.in/dedis/kyber.v1/group/mod"
+	"gopkg.in/dedis/kyber.v1/util/bytes"
+	"gopkg.in/dedis/kyber.v1/util/marshalling"
+	"gopkg.in/dedis/kyber.v1/util/random"
+	"gopkg.in/dedis/kyber.v1/util/subtle"
+)
+
 // This code is a port of the public domain, "ref10" implementation of ed25519
 // from SUPERCOP.
 
```

Comment:
Add reference to supercop:

https://bench.cr.yp.to/supercop.html
and perhaps:
https://github.com/anonimal/supercop-ed25519-ref10